frontend/src/components/uam/helpers/get-columns-helper/get-columns.helper.ts

```diff
@@ -0,0 +1,27 @@
+import { UsersGetAllItemResponseDto } from 'common/types/types';
+import { Column } from 'react-table';
+
+import { UserTableAccessor, UserTableHeader } from '../../common/enums/enums';
+
+const getColumns = (): Column<UsersGetAllItemResponseDto>[] => {
+  return [
+    {
+      Header: UserTableHeader.ID,
+      accessor: UserTableAccessor.ID,
```

Comment:
```suggestion
      Header: UserTableHeader.ID,
      accessor: UserTableHeader.ID,
```
you can avoid the second one and use only one for it